docker/frontend/Dockerfile

```diff
@@ -25,7 +25,7 @@ COPY --from=git /CodeDefenders /CodeDefenders
 WORKDIR /CodeDefenders
 
 # Build Code Defenders for Docker using config.properties@docker
-RUN cp config.properties@docker config.properties && \
+RUN cp config.properties@docker.example config.properties && \
```

Comment:
I renamed/moved the config.properties* files, so it's not that easy to accidental commit secrets to the repository.
If `config.properties@docker` doesn't contain important secrets maybe we should just rename it to the old filename and remove it from the .gitignore?
The other option would be to state in the Readme to first copy `config.properties@docker.example` to `config.properties@docker`

Reply:
The content of this file could be generated in a similar way to tomcat-users.xml / context.xml. I can update the PR to generate it using environment variables if you want.

Reply:
Ok, I had a look to the config.properties and pom.xml. Seems that the it is used at build time, however seems that most of the configuration parameters (db.*, cluster.*) are replaced in the context.xml file. @ein-stein-chen could you confirm my findings so I can update the PR accordingly ?

Reply:
I am not familiar with our docker deployment yet.
As far as I understand it, the `context.xml` file is generated from the `config.properties` file at build time.
So I guess we don't need `config.properties` at all, except it's existence is enforced from the build script.
When generating `config.properties` or `context.xml` it would be nice if we could reuse the environment variables of the database container for the `db.*` properties. `data.dir` and `ant.home` should be fixed due to the container. I am not sure if we can use slurm with docker but this would make the `cluster.*` properties fixed as well. The other four could be generated using environment variables.

Reply:
I've updated de PR so it is possible to dynamically fill the context.xml file using environment variables. I've tested it in a internal CodeDefenders deployment. I did not test SLURM configuration, although there is a project [SciDAS/slurm-in-docker](https://github.com/SciDAS/slurm-in-docker) that looks promising to deploy SLURM using docker too.